### PR TITLE
Fix session scroller

### PR DESCRIPTION
**Pull request by kimlisa**
fixes https://github.com/gravitational/webapps/issues/96

#### Description
- Disable terminal scrolling for session replay's so that users rely on player controller
- Extend a xterms `_getMouseBufferCoords` to calculate offset between child and parent viewport containing the terminal.

#### Testing
Manually tested on linux chrome and firefox. 

Session replay with browser window smaller than terminal (note the aligned text selection):
Peek 2020-06-09 01-52

### CODE REVIEW

File: packages/teleport/src/console/Console.tsx [hunk 1]
```diff
@@ -117,4 +117,8 @@ const StyledConsole = styled.div`
   top: 0;
   display: flex;
   flex-direction: column;
+
+  .terminal .xterm-viewport {
```
Comment (alex-kovoy) [inline packages/teleport/src/console/Console.tsx:121]: this has been added in the last commit

Comment (kimlisa) [reply packages/teleport/src/console/Console.tsx]: removed

File: packages/teleport/src/lib/term/terminal.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ class TtyTerminal {
       cursorBlink: false,
     });
 
-    this.term.open(this._el);
+    this.term.open(this._el, true);
```
[the remark this answers is not in the document]
Comment (alex-kovoy) [reply packages/teleport/src/lib/term/terminal.js:57]: Your branch is outdated. It has been fixed in master.

File: packages/teleport/src/player/Xterm/Xterm.tsx [hunk 3]
```diff
@@ -28,6 +28,12 @@ export default function Xterm({ tty }: { tty: Tty }) {
     });
 
     term.open();
+    term.extendGetBufferCoords(refContainer);
+
+    term.tty.on('terminal.data', () => {
```
Comment (alex-kovoy) [inline packages/teleport/src/player/Xterm/Xterm.tsx:33]: There is an enum for 'terminal.data'. Lets use that instead of hardcoded string.

Comment (kimlisa) [reply packages/teleport/src/player/Xterm/Xterm.tsx]: revised: https://github.com/gravitational/webapps/pull/99/commits/868b747bdb286ec6d6ef2bf64b529a11fe252d6b

File: packages/teleport/src/lib/term/terminal.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kimlisa) [reply packages/teleport/src/lib/term/terminal.js]: removed: https://github.com/gravitational/webapps/pull/99/commits/868b747bdb286ec6d6ef2bf64b529a11fe252d6b

[the remark this answers is not in the document]
Comment (kimlisa) [reply packages/teleport/src/lib/term/terminal.js]: merged master unto this branch: https://github.com/gravitational/webapps/pull/99/commits/9f37dbdaf53fa0d6665f219317ecc66c19b99a65

**Pull request merged**